docker-ee-for-windows/install.md

```diff
@@ -0,0 +1,115 @@
+
+> **Already have Docker EE for Windows?** If you already have Docker EE for
+Windows installed, and you're ready to get started, skip to
+[Get started with Docker for Windows](/docker-for-windows/index.md) for a quick tour of the command line, settings, and tools.
+>
+>**Looking for Release Notes?** [Get release notes for all
+versions here](https://docs.docker.com/release-notes/) or subscribe to the [ releases feed on the Docker Blog](http://blog.docker.com/category/engineering/docker-releases/).
+
+## Install Docker EE
+
+Docker EE for Windows requires Windows Server 2016. Please see [What to know before you install](/docker-ee-for-windows/#what-to-know-before-you-install) for a full list of prerequisites. To install Docker Community Edition (*Docker CE*) on a Windows 10 machine, see [Install Docker for Windows](/docker-for-windows/install.md).
+
+1.  Open a PowerShell command prompt, and type the following commands.
+
+    ```bash
+    Install-Module -Name DockerMsftProvider -Force
+    Install-Package -Name docker -ProviderName DockerMsftProvider -Force
+    Restart-Computer -Force
+    ```
+
+2.  Test your Docker EE installation by running the `hello-world` container.
+
+    ```bash
+    docker run hello-world:nanoserver
+    ```
+
+## Install Docker EE on an offline machine
+
+Use the following steps when you want to install Docker EE on a machine that
+isn't connected to the internet.
+
+1.  In a PowerShell command prompt, download the installer archive on a machine that has a connection.
+
+    ```bash
+    # on online machine
+    invoke-webrequest -UseBasicparsing -Outfile docker.zip https://download.docker.com/components/engine/windows-server/17.03/docker-17.03.0-ee.zip
+    ```
+
+2.  Copy the zip file to the machine that isn't connected, by using a thumb drive or by intranet. In a PowerShell command prompt, use the following commands to extract the archive, register, and start the Docker service.
+
+    ```bash
+    #— extract the archive —
+    Expand-Archive docker.zip -DestinationPath $Env:ProgramFiles
+    #clean up zip
+    Remove-Item -Force docker.zip
+    # enable hyper-v if not yet enabled
+    Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All
+
+    #adding docker to PATH
+    $env:path += "$env:ProgramFiles\Docker"
+
+    #register docker daemon as a service
+    dockerd --register-service
+
+    #start the daemon
+    Start-Service docker
+    ```
+
+3.  Test your Docker EE installation by running the `hello-world` container.
+
+    ```
+    docker run hello-world:nanoserver
+    ```
+
+## Install Docker EE by using OneGet
+
+If you want to install Docker EE by using [OneGet](https://github.com/oneget/oneget), follow the steps described in [Windows Containers on Windows Server](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/quick-start-windows-server).
+
+##  What to know before you install
+
```

Comment:
Does Docker EE on Server require Hyper-V? If so, should I include the notes from https://docs.docker.com/docker-for-windows/install/#what-to-know-before-you-install ?